### PR TITLE
Added delete-complete-files properties

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
cnp-flux-config: https://github.com/hmcts/cnp-flux-config/pull/2866
Cron expression is to run this job once per day at 8pm. TBD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by apereverzin** [comment]
Fixed

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -190,7 +190,8 @@ scheduling:
       cron: ${DELETE_REJECTED_FILES_CRON}
       ttl: ${DELETE_REJECTED_FILES_TTL}
     delete-complete-files:
-      enabled: false
+      enabled: ${DELETE_COMPLETE_FILES_ENABLED:false}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml:193]: can we remove default ?

Comment (apereverzin) [reply src/main/resources/application.yaml:193]: having default value makes it bullet proof. other properties have default values. why do you suggest removing default?

Comment (doncem) [reply src/main/resources/application.yaml:193]: there are some conflicts in bootstrapping those application configs in during flux deployments on different environments. the working solution is to remove defaults

**Comment by apereverzin** [comment]
PR-1142

**Pull request closed without merging**